### PR TITLE
Document Signed APIs and Providers

### PR DESCRIPTION
**Pull request by Ashar2shahid**
Documenting Signed APIs based on #831 and also documenting new providers

### CODE REVIEW

**Review by wkande — APPROVED** [review]
LGTM

File: docs/reference/dapis/understand/signed-apis.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+<PageHeader/>
+
+# Signed APIs
+
+Signed APIs store the data pushed by Airnodes and expose them to the public via
+an API. This allows for various use cases, including powering API3 push oracle.
+
+Both the Airnode and Signed API implementations are
+[open source](https://github.com/api3dao/signed-api) to increase the
+transparency and security of the process.
```
Comment (dcroote) [inline docs/reference/dapis/understand/signed-apis.md:20]:
```suggestion
Both the [Airnode](https://github.com/api3dao/airnode) and
[Signed API](https://github.com/api3dao/signed-api) implementations are open
source to increase the transparency and security of the process.
```

File: docs/reference/dapis/understand/signed-apis.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+outline: deep
+tags: signed-apis
+---
+
+<PageHeader/>
+
+# Signed APIs
+
+Signed APIs store the data pushed by Airnodes and expose them to the public via
+an API. This allows for various use cases, including powering API3 push oracle.
```
Comment (dcroote) [inline docs/reference/dapis/understand/signed-apis.md:16]:
```suggestion
an API. This allows for various use cases, including powering API3 push oracles.
```

File: docs/reference/dapis/understand/signed-apis.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+[open source](https://github.com/api3dao/signed-api) to increase the
+transparency and security of the process.
+
+### Endpoints
+
+Signed APIs are also open sourced for anyone to use. This is yet another step
+towards more decentralization, because even if API3 oracle service is down,
+anyone can use these existing Signed APIs to do the updates instead. This also
+allows MEV searchers to utilize the data to perform updates outside of the
+deviation parameters to further increase real-timeness of the data feeds.
```
Comment (dcroote) [inline docs/reference/dapis/understand/signed-apis.md:28]:
```suggestion
deviation parameters to further increase data feed currentness.
```

**Review by dcroote — APPROVED** [review]
Nice. A few minor suggestions.

**Review by Siegrift — APPROVED** [review]
Thanks Ashar, 👍 LGTM

**Pull request merged**